### PR TITLE
Server db schema update

### PR DESCRIPTION
**Pull request by itsnormsyo**
Server db schema update. Waiting for confirmation from @jessicaawhy and @finchzebra  before merge

### CODE REVIEW

File: client/components/Tracker/Form/Form.jsx [hunk 1]
```diff
@@ -1,11 +1,21 @@
...
 const Form = () => {
   const [feeling, updateFeeling] = useState('');
   const [thoughts, updateThoughts] = useState('');
 
   const handleSubmit = (event) => {
+    //(user_id, thought, feeling_id, track_date)
+
+    const trackDate = new Date(Date.now()).toLocaleString().split(',')[0].split('/').join('-');
+
+    axios.post('/tracker/postListItem', {
```
Comment (jessicaawhy) [inline client/components/Tracker/Form/Form.jsx:14]: Consider /tracker/list for a restful implementation

File: client/components/Tracker/Form/Form.jsx [hunk 2]
```diff
@@ -1,11 +1,21 @@
 import React, { useState, useEffect } from 'react';
+import axios from 'axios';
 import styled from 'styled-components';
 
 const Form = () => {
   const [feeling, updateFeeling] = useState('');
   const [thoughts, updateThoughts] = useState('');
 
   const handleSubmit = (event) => {
+    //(user_id, thought, feeling_id, track_date)
```
Comment (jessicaawhy) [inline client/components/Tracker/Form/Form.jsx:10]: I usually prevent the default action first

File: server/controllers/trackerController.js [hunk 3]
```diff
@@ -15,3 +16,37 @@ exports.exampleGet = async (req, res) => {
...
+  console.log(reqParams);
+  console.log('I am here in the post list item controller');
+  const query = `
+    INSERT INTO tracker (user_id, thought, feeling_id, track_date) VALUES (2, $1, $2, $3)
+  `;
+
+  try {
+    const result = await queryDB(pool, query, reqParams);
+
+    res.sendStatus(204);
```
Comment (jessicaawhy) [inline server/controllers/trackerController.js:48]: I think status code 201 - created would be appropriate

File: client/components/Tracker/Form/Form.jsx [hunk 4]
```diff
@@ -1,11 +1,21 @@
...
+
+    const trackDate = new Date(Date.now()).toLocaleString().split(',')[0].split('/').join('-');
+
+    axios.post('/tracker/postListItem', {
+      feeling, thoughts, trackDate,
+    })
+      .then(() => console.log('succesful post to the DB'))
```
Comment (jessicaawhy) [inline client/components/Tracker/Form/Form.jsx:17]: To consider later - rendering out the latest post if successful?

**Review by jessicaawhy — APPROVED** [review]

File: client/components/Tracker/Form/Form.jsx (hunk 4 above)
Comment (itsnormsyo) [reply client/components/Tracker/Form/Form.jsx:17]: Sounds good! going to implement in a separate branch!

File: client/components/Tracker/Form/Form.jsx (hunk 1 above)
Comment (itsnormsyo) [reply client/components/Tracker/Form/Form.jsx:14]: updated! thanks!

File: client/components/Tracker/Form/Form.jsx (hunk 2 above)
Comment (itsnormsyo) [reply client/components/Tracker/Form/Form.jsx:10]: Updated! Thanks!

**Pull request merged**